hoover/contrib/__init__.py

```diff
@@ -2,4 +2,5 @@
 
 class installed:
     twofactor = ('hoover.contrib.twofactor' in settings.INSTALLED_APPS)
+    oauth2 = ('hoover.contrib.oauth2' in settings.INSTALLED_APPS)
```

Comment:
Unrelated aspect: one would have to either modify `common.py` or overwrite it in `local.py`. Modifying `common.py` for each instance causes `git` to complain.

How do we document this problem? Do we invite users to copy/paste the whole `INSTALLED_APPS` list to `local.py`?